### PR TITLE
api/enforcement: Add endpoint for operators

### PR DESCRIPTION
**Pull request by suutari-ai**
Add API endpoint for listing the operators.

Also add operator identifier to the valid_parking endpoint so that it is
possible to link them properly (not just by name).

### CODE REVIEW

File: parkings/api/enforcement/urls.py [hunk 1]
```diff
@@ -1,9 +1,11 @@
 from django.conf.urls import include, url
 from rest_framework.routers import DefaultRouter
 
+from .operator import OperatorViewSet
 from .valid_parking import ValidParkingViewSet
 
 router = DefaultRouter()
+router.register('operator', OperatorViewSet, base_name='operator')
```
Comment (tuomas777) [inline parkings/api/enforcement/urls.py]: `base_name` not needed here.

File: parkings/tests/api/enforcement/test_valid_parking.py [hunk 2]
```diff
@@ -75,7 +75,7 @@ def check_parking_data_keys(parking_data):
         'id', 'created_at', 'modified_at',
         'registration_number',
         'time_start', 'time_end', 'zone',
-        'operator_name',
+        'operator', 'operator_name',
```
Comment (tuomas777) [inline parkings/tests/api/enforcement/test_valid_parking.py]: Could also check the contents of this field.

File: parkings/api/enforcement/urls.py (hunk 1 above)
Comment (suutari-ai) [reply parkings/api/enforcement/urls.py]: I like the explicit base_name more. It's kind of "magic" when its determined from the model behind the serializer's queryset.

File: parkings/tests/api/enforcement/test_valid_parking.py (hunk 2 above)
Comment (suutari-ai) [reply parkings/tests/api/enforcement/test_valid_parking.py]: Right. Does it now.

**Pull request merged**